framework/wazuh/core/agent.py

```diff
@@ -1245,7 +1250,7 @@ def calculate_status(last_keep_alive, pending, today=datetime.utcnow()):
         return "disconnected" if difference > common.limit_seconds else ("pending" if pending else "active")
 
 
-def send_restart_command(agent_id: str = '', agent_version: str = '') -> str:
+def send_restart_command(agent_id: str = '', agent_version: str = '', wq: WazuhQueue = None) -> str:
```

Comment:
This function will not work on its own with its default parameter values. Consider adding extra logic or simply make these parameters positional. The latter would cancel the previous requested changes.

Reply:
Parameters of `send_restart_command` and `Agent.restart` are now positional